qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/common/data_handling/test/test_data_presenter.py

```diff
@@ -43,6 +43,17 @@ def setUp(self):
         self.ws3 = mock.MagicMock()
         self.ws3.getAxis.return_value = mock_axis_d
 
+    def _setup_fitting_table_rows(self, ws_names, active_ws_xunit=None):
+        self.presenter.row_numbers = data_presenter.TwoWayRowDict()
+        for i, val in enumerate(ws_names):
```

Comment:
FYI I prefer more meaningful variable names in places like this
`for iws, ws_name in enumerate(ws_names)`
but it's fine as it is given it's only a helper method in a unit test file (just for future reference).
Elsewhere all the variable names seem good to me!

Reply:
Thanks for pointing out. This was changed